### DISCUSSION TITLE
Adapt STEMMUS_SCOPE to BMI

### DISCUSSION THREAD

**Discussion by MostafaGomaa93** [EcoExtreML/STEMMUS_SCOPE#205] [General]
Hi @SarahAlidoost 
Thanks for your suggestion that we open a discussion here (actually I didn't know about this option before). My question is about the BMI. I am supposed to couple STEMMUS_SCOPE to the groundwater model (MODFLOW 6) via the MODFLOW API package (described [here](https://www.sciencedirect.com/science/article/pii/S1364815221002991)). So, I would like to know when roughly the STEMMUS_SCOPE model will be adapted to the BMI and be ready to be coupled with MODFLOW.

**Comment by SarahAlidoost** [comment]
> Hi @SarahAlidoost Thanks for your suggestion that we open a discussion here (actually I didn't know about this option before). My question is about the BMI. I am supposed to couple STEMMUS_SCOPE to the groundwater model (MODFLOW 6) via the MODFLOW API package (described [here](https://www.sciencedirect.com/science/article/pii/S1364815221002991)). So, I would like to know when roughly the STEMMUS_SCOPE model will be adapted to the BMI and be ready to be coupled with MODFLOW.

Good question. The general plan is to start the BMI implementation after [zampy](https://github.com/EcoExtreML/zampy) and [emulator](https://github.com/EcoExtreML/Emulator). The details of the project planning are usually discussed in our biweekly meetings, the next one is on 28th November, 10:00-11:00 CET. If you are interested to know more about it, please join the meeting. You can ask @yijianzeng to share the meeting info with you. 
The BMI implementations would be mainly in Python, with some adjustments to the model in MATLAB. The implementation can benefit from your input and knowledge. If you are interested in contributing to the development of BMI, please let me know. We can further discuss the plan in the biweekly meeting.

**Comment by MostafaGomaa93** [comment]
Okay, thanks for the info. I will ask @yijianzeng for the meeting info and will try to join.
Mostafa
